app/jobs/fca_confirmation_job.rb

```diff
@@ -1,21 +1,25 @@
 require 'digest/sha1'
-$LOAD_PATH.unshift(File.join(Rails.root, 'lib'))
+$LOAD_PATH.unshift(Rails.root.join('lib'))
 require 'cloud'
 require 'fca'
 
 class FcaConfirmationJob < ActiveJob::Base
   include Sidekiq::Worker
+
+  unique_args = ->(args) { Digest::SHA1.hexdigest(args.first.to_s) }
   sidekiq_options unique:      :until_executed,
-                  unique_args: ->(args) { Digest::SHA1.hexdigest(args.first.to_s) }
+                  unique_args: unique_args
   queue_as :default
 
   def perform(import_job_id)
-    if import = FcaImport.find_by_id(import_job_id) # rubocop:disable all
+    # rubocop:disable all
```

Comment:
what are the failing cops in this block that requires all of rubocop to be disabled?

Reply:
@willhall88 There were a few, and we were reluctant to refactor this too much as we weren't sure how thorough the test coverage is for this class. A few of them are fairly trivial though, so I've pushed some more fixes. Now only one cop is explicitly disabled (which I don't think we can fix without potentially altering the behaviour).